sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/QueryBase.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.DigitalTwins.Core.QueryBuilder;
+
+namespace Azure.DigitalTwins.Core.Queries.QueryBuilders
+{
+    /// <summary>
+    /// Parent class for all query clauses.
+    /// </summary>
+    public abstract class QueryBase
+    {
+        /// <summary>
+        /// Finalizes query.
```

Comment:
In c# a finalizer is a method like a ctor that runs when the class goes away. For that reason, I'd stay away from the word "finalizes" because it may make something think of that.